redfish/rendered_rules.go

```diff
@@ -2900,7 +3085,7 @@ var Rules = map[string]*CollectRule{
 					},
 					{
 						Pointer: "/OutputPowerWatts/Reading",
-						Name:    "chassis_powersubsystem_powersupplies_metrics_outputpowerwatts_reading",
+						Name:    "chassis_powersubsystem_powersupplies_metrics_outputpowerwatts_readin",
```

Comment:
Something went wrong.
This diff should not be generated just by re-generating all.

The source rule file is corrupted.
https://github.com/cybozu-go/setup-hw/pull/121/files#diff-091f87b6e75e6020d6e63298a2c62f4da0bf9d2b5925ec832fa9de9a45fcbcf9R166
Please **regenerate** this source file by running `collector generate-rule` or `base-rules/run_generate-rule.sh`, and make another pull request before adding `1.19.0` tag.

Reply:
@takara9 
If it's hard to generate the rule file `dell_redfish_1.20.1.yml`, just copy `dell_redfish_1.18.0.yml` onto `dell_redfish_1.20.1.yml` without modification.

In the normal manner, we need to retrieve the Redfish responses by executing `collector show` on our seed servers.